samples/Azure.Storage.Tables/Azure.Storage.Tables/CodeModel.yaml

```diff
@@ -1112,7 +1112,7 @@ globalParameters:
   protocol: !<!Protocols> 
     http: !<!HttpParameter> 
       in: header
-- !<!Parameter> &ref_79
```

Comment:
This one is interesting. Considering you didn't update the autorest why did this change.

Reply:
🤔